tests/add-delete-add.spec.ts

```diff
@@ -1,9 +1,18 @@
-import { test, expect } from "@playwright/test";
+import { test, expect, type Page } from "@playwright/test";
 
-test("testing adding interventions, then deleting one, then adding another", async ({
-  page,
-}) => {
-  await page.goto("/scheme.html?authority=Derby#16.84/52.906457/-1.504519");
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
```

Comment:
Is there a way we can share this `beforeAll` code over multiple files? Not sharing workers between files, just deduping the code. If we have a `shared.ts` file and import from it, does that work? On the "fixtures" docs, they say

> Fixtures are reusable between test files - you can define them once and use in all your tests. That's how Playwright's built-in page fixture works.

Which makes me worry this is somehow not straightforward for hooks.

But this shouldn't block merging this PR, since it's an improvement in stability

Reply:
This looks fine to me, and is nice that it's explicit we're using `beforeAll` in each test file. Thanks!